src/main/java/com/adobe/commerce/cif/api/HealthApi.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ *
+ *    Copyright 2018 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+package com.adobe.commerce.cif.api;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+import static com.adobe.commerce.cif.api.Constants.HTTP_OK;
+import static com.adobe.commerce.cif.api.Constants.HTTP_OK_MESSAGE;
+import static com.adobe.commerce.cif.api.Constants.HTTP_SERVICE_UNAVAILABLE;
+import static com.adobe.commerce.cif.api.Constants.HTTP_SERVICE_UNAVAILABLE_MESSAGE;
+
+@Path("/_health")
+@Api(value = "/_health")
+@Produces(MediaType.TEXT_PLAIN)
+public interface HealthApi {
+
+    @GET
+    @Path("/")
+    @ApiOperation(
+        value = "Returns the health status.",
+        notes = "This endpoint is used to check if underlying services are healthy. This is an internal API used for monitoring and must "
+                + "not be used by any storefront and must not exposed to any end customers. The endpoint must not return any information in"
```

Comment:
`must not exposed to any end customers` how should that be achieved? I think that is not doable today. If deployed as runtime action it is visible.

Reply:
"end customers" are the ones that are using the store front.

Reply:
I disagree. I think all modern websites should have a status page. If something is wrong with the site, the storefront can display a notification somewhere on the page telling something like: `Adding to cart is affected by a problem and we are working on fixing it. For more information check: link to status page`.

I think the health check API should be public and if storefronts want to use, they should be able to do this.